starboard/extension/loader_app_metrics.h

```diff
@@ -46,15 +46,47 @@ typedef struct StarboardExtensionLoaderAppMetricsApi {
   // This specifies the version of the API that is implemented.
   uint32_t version;
 
-  // The fields below this point were added in version 1 or later.
-
   // The accessors and mutators below are assumed to be called from the same
   // thread: Cobalt's application thread.
 
-  void (*SetCrashpadInstallationStatus)(CrashpadInstallationStatus status);
+  // The fields below this point were added in version 1 or later.
 
+  void (*SetCrashpadInstallationStatus)(CrashpadInstallationStatus status);
   CrashpadInstallationStatus (*GetCrashpadInstallationStatus)();
 
+  // The fields below this point were added in version 2 or later.
```

Comment:
We heaven't release this extension yet so maybe you don't need to bump the version.

Reply:
Thanks, yeah, I figured bumping the version would make the cherry picking to the LTS branch more carefree but let me know if you think it's worth keeping the extension at version 1 to keep the code cleaner - I don't feel strongly.